docs/configuring-playbook-bridge-mautrix-discord.md

```diff
@@ -0,0 +1,43 @@
+# Setting up Mautrix Discord (optional)
+
+The playbook can install and configure [mautrix-discord](https://github.com/mautrix/discord) for you.
```

Comment:
```suggestion
**Note**: bridging to [Discord](https://discordapp.com/) can also happen via the [mx-puppet-discord](configuring-playbook-bridge-mx-puppet-discord.md) and [matrix-appservice-discord](configuring-playbook-bridge-appservice-discord.md) bridges supported by the playbook.

The playbook can install and configure [mautrix-discord](https://github.com/mautrix/discord) for you.
```

Updating `configuring-playbook-bridge-mx-puppet-discord.md` and `configuring-playbook-bridge-appservice-discord.md` in a similar way would be good. I have not tested the mautrix-discord bridge, but I believe it would be the best one of them all and we should probably even recommend it more strongly than the others.

Reply:
If you call the mx-puppet-discord, then you should tell that there is no maintainer for it anymore....
https://matrix.to/#/!uOjvskgRfNgksdiArw:sorunome.de/$wRt1v3lcrt9RbmkRzqidlz9doEcdXyv1r2iDnAIn8mM?via=matrix.org&via=privacytools.io&via=tchncs.de